frontend/src/scenes/onboarding/MDX.tsx

```diff
@@ -0,0 +1,329 @@
+import { MDXProvider } from '@mdx-js/react'
```

Comment:
Another thing that sucks is that MDX works at build time, and it is an entirely different set of things to implement than plain markdown.

Mostly, it just lets you pass in components and render them during build like templating.

The problem is that it's compilation is very restrictive it seems, and finicky